src/Tasks/MigrateContentToElement.php

```diff
@@ -49,12 +51,14 @@ class MigrateContentToElement extends BuildTask
      */
     private static $publish_changes = true;
 
-    protected $title = 'MigrateContentToElement';
+    protected static string $commandName = 'elemental-migrate-content';
```

Comment:
```suggestion
    protected static string $commandName = 'MigrateContentToElement';
```

Reply:
Discussion about this and other task command names is being held in the main issue.